### ISSUE TITLE
Unable to open doors in mouselook when wearing a collar

### ISSUE THREAD

**Issue by Evilgertie** [OpenCollarTeam/OpenCollar#747]
What version of OpenCollar are you using?
8.2.1000

What behaviour did you expect? 
In mouselook, click on a door and it should open (assuming it can be opened with a click in third person).

What behaviour did you see instead? 
When wearing a collar you can't open a door in mouselook.  You can still open it in 3rd person with the collar on.

What steps does someone need to take to reproduce the problem?
Just wear a new collar out of the box.  Go into mouselook and try and open a door.  Use SL viewer or Firestorm.  You don't even need RLV active.

**Comment by Medea-Destiny** [comment]
Likely caused by the llTakeControls bug interfering with mouse clicks in mouselook mode. oc_anim takes controls over-aggressively and this should be looked into as the potential source of the observed issue. 

In the first instance we could simply try removing CONTROL_ML_LBUTTON from the list of controls taken in the oc_anim script and see if this fixes the problem; I can't think of any reason why we need to take that control anyway, off the top of my head (maybe click to movement?) Longer term oc_anim could really do with an efficiency refactoring.

**Comment by Evilgertie** [comment]
Hopefully something can be done.  I can't be the only person that likes to use first person view and get through doors.
I've just had a test with some old collars (3.4, 3,5 and 3.6) no issues there.  I don't have any other collars so can't pinpoint when this behaviour started.

**Comment by K9K8E** [comment]
I also noticed this after updating my 3.997 collar to 8.2.1. It's line 469 of oc_anim that causes it. Once removed, left clicking in mouselook is fine.
(No, Gertie, you're not the only one, there are two of us! But I think that is about it, OC chat was a hopeless mess of unhelpful and sometimes rude comments or 'I dun get it???' but one person IMed me and with his help I found the issue.  Many people stated that it was normal, since they couldn't ML left click even without a collar, hinting that perhaps this scripting glitch is commonplace in other products)
Thank you OC peoples for all your continued hard work <3

**Comment by Evilgertie** [comment]
Thank you so much this does fix the issue.  I had a bit of trouble figuring out the right way to edit the script as I usually edit scripts in place but I got there in the end.  I'll be making an instructional notecard if anyone contacts me inworld (I'm the only Evilgertie).
You are quite correct about other products generating the same issue, I have an alt with a Genus head which has this issue.  I caused wonderful confusion bringing it up in that group.  I didn't mention it here before as I didn't want to muddy the waters.  Hopefully the next update will either have this line deleted or maybe an explanation of why it's there.  I guess I'm a tester for now as I have an updated script.
